deployment/helm/node-feature-discovery/templates/cert-manager-certs.yaml

```
@@ -0,0 +1,66 @@
+{{- if .Values.tls.certManager }}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nfd-master-cert
+spec:
+  secretName: nfd-master-cert
+  subject:
+    organizations:
+    - node-feature-discovery
+  commonName: nfd-master
+  dnsNames:
+  # must match the service name
+  - {{ include "node-feature-discovery.fullname" . }}-master
+  # first one is configured for use by the worker; below are for completeness  
```

nit: trailing whitespace

```suggestion
  # first one is configured for use by the worker; below are for completeness
```